picard/config.py

```diff
@@ -371,14 +371,15 @@ class Option(QtCore.QObject):
     registry = {}
     qtype = None
 
-    def __init__(self, section, name, default, title=None):
+    def __init__(self, section, name, default, title=None, highlight=None):
         key = (section, name)
         if key in self.registry:
             raise OptionError("Already declared", section, name)
         super().__init__()
         self.section = section
         self.name = name
         self.default = default
+        self.highlight = highlight
         self.title = title
```

Comment:
If there is a title set, and highlight is None, perhaps we could set self.highlight to an empty list?  Something like:

```suggestion
        self.title = title
        self.highlight = highlight
        if self.title is not None and not self.highlight:
            self.highlight = []
```

That way we may be able to simply avoid setting the highlight attribute when adding an option to an option page, rather than having to include the setting `highlight=[]`.  Even though there aren't very many where this would apply, this might simplify entering an option on one of the pages.  If there is no highlight specified, then nothing will be highlighted.